homeassistant/components/stream/worker.py

```diff
@@ -92,12 +100,17 @@ def reset_av(self):
                 template=self._input_audio_stream
             )
 
-    def set_streams(self, video_stream, audio_stream):
+    def set_streams(
+        self,
+        video_stream: av.video.VideoStream,
+        audio_stream,
+        # no type hint for audio_stream until https://github.com/PyAV-Org/PyAV/pull/775 is merged
```

Comment:
Well, it still can be imported as `av.audio.stream.AudioStream`.

Reply:
Really? I tried it before and it didn't seem to work